### PR TITLE
various update about 'syntax sections'

### PR DESCRIPTION
**Pull request by jasonren0403**

### CODE REVIEW

File: files/en-us/mdn/writing_guidelines/page_structures/syntax_sections/index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The syntax section of an MDN reference page contains a syntax box defining the e
 
 Syntax sections for API reference pages are written manually, and may differ slightly based on the feature being documented.
 The section starts with a heading (typically level two heading `##`) named "Syntax", and must be included at the top of the reference page (just below the introductory material).
-Below the heading is a code block showing the feature's exact syntax, demarcated using code fence ` ``` [markup-language] ` class.
+Below the heading is a code block showing the feature's exact syntax, demarcated using code fence ` ```[markup-language] ` class.
```
Comment (jasonren0403) [inline files/en-us/mdn/writing_guidelines/page_structures/syntax_sections/index.md:15]: Currently we are not putting spaces between three-backticks and markup-language notation.

File: files/en-us/mdn/writing_guidelines/page_structures/syntax_sections/index.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ A few rules to follow in terms of markup within the syntax block:
 
   ```js-nolint
   querySelector(selector)
-  //responseStr = element.querySelector(selector)
+  // responseStr = element.querySelector(selector)
```
Comment (jasonren0403) [inline files/en-us/mdn/writing_guidelines/page_structures/syntax_sections/index.md:39]: align with example just below this line

File: files/en-us/mdn/writing_guidelines/page_structures/syntax_sections/index.md [hunk 3]
```diff
@@ -183,7 +183,7 @@ splice(start, deleteCount, item1, item2, /* …, */ itemN)
 
 Next, include a "Parameters" subsection, which explains what each parameter should be, in a description list. Parameters that are objects containing multiple members can include a nested description list, which itself includes an explanation of what each member should be. Optional parameters should be marked with an \\{{optional_inline}} macro call next to their name in the description term.
 
-The name of each parameter in the list should be contained in a {{HTMLElement("code")}} block.
+The name of each parameter in the list should be contained in markdown code fence notation `` ` ` ``.
```
Comment (jasonren0403) [inline files/en-us/mdn/writing_guidelines/page_structures/syntax_sections/index.md:186]: We are putting current code contents into markdown backticks, not the HTML `<code>`

Comment (hamishwillee) [reply files/en-us/mdn/writing_guidelines/page_structures/syntax_sections/index.md:189]: Yes, this is good. In a future iteration we might put examples in this section.

**Review by hamishwillee — APPROVED** [review]
Thanks very much - looks good to me. I added a note about the use of `-nolint` - it seems relevant given we refer to the text as 'markdown-language'.

**Pull request merged**